File: backend/src/model/comment.js
module.exports = (sequelize, DataTypes) => {
  const Comment = sequelize.define('Comment', {
    id: {
      type: DataTypes.INTEGER,
      autoIncrement: true,
      primaryKey: true,
    },
    content: {
      type: DataTypes.TEXT,
      allowNull: false,
    },
    timestamp: {
      type: DataTypes.DATE,
      defaultValue: DataTypes.NOW,
    },
  });
  Comment.associate = (db) => {
    db.Comment.belongsTo(db.User, {
      foreignKey: 'user_id',
    });
    db.Comment.belongsTo(db.Issue, {
      foreignKey: 'issue_id',
    });
  };
  return Comment;
};

File: iOS/IssueTracker/IssueTracker/Models/Comment.swift
//
//  Comment.swift
//  IssueTracker
//
//  Created by 현기엽 on 2020/11/02.
//

import Foundation

struct Comments: Codable {
    let comments: [Comment]
}

struct Comment: Model, Codable {
    let id: Int
    let content: String
    let timestamp: String
    let user_id: Int
    let issue_id: Int?
}
